react/components/SliderTrack.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { FC } from 'react'
+import React, { FC, Fragment } from 'react'
+import { useSSR } from 'vtex.render-runtime/core/main'
```

Comment:
hmmm not sure about this.

Importing a module that is not directly exposed seems easy to break

can you import like 
```
import { useSSR } from 'vtex.render-runtime'
```

Reply:
Nice catch! I did not think about how that could break in the future.
Also, nope, I can't... for some reason, a lot of stuff is not exported through just `import {  } from 'vtex.render-runtime'`. I was trying to use the typedefs from running `vtex setup`.
I'll just use a local definition as we normally do :)